src/txmempool.h

```diff
@@ -127,6 +149,11 @@ class CTxMemPoolEntry
     CAmount GetModFeesWithAncestors() const { return nModFeesWithAncestors; }
     int64_t GetSigOpCostWithAncestors() const { return nSigOpCostWithAncestors; }
 
+    const Parents& GetMemPoolParentsConst() const { return m_parents; }
+    const Children& GetMemPoolChildrenConst() const { return m_children; }
+    Parents& GetMemPoolParents() const { return m_parents; }
+    Children& GetMemPoolChildren() const { return m_children; }
```

Comment:
It's C++ convention that the const one has a const-y name (e.g., begin -> cbegin). I opted to suffix with const rather than CGet... because we use the hungarian notation everywhere so it would look like a class.

Future work can also get rid of the mutating getters use entirely by moving UpdateChild into the CTXMemPoolEntry itself. This isn't done here as it is a bigger refactor (requires changing the interfaces) and has no functional benefit.

Reply:
Makes sense. Closing discussion